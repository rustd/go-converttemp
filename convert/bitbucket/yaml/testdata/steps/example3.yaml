# Example using the script property to run a sequence of commands for a single step

pipelines:
  default:
    - step:
        script:
          - echo "Hello,"
          - echo "World!"
          - ./build.sh
