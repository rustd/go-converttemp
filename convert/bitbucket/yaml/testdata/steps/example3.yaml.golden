{
    "pipelines": {
        "default": [
            {
                "step": {
                    "script": [
                        "echo \"Hello,\"",
                        "echo \"World!\"",
                        "./build.sh"
                    ]
                }
            }
        ]
    }
}